src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/JmapEmail.scala

```diff
@@ -114,4 +118,59 @@ object JmapEmail {
            |    "c1"]]
            |}""".stripMargin))
   }
+
+  def sendMessages(recipientFeeder: RecipientFeederBuilder): ChainBuilder = {
+    val mailFeeder = Iterator.continually(
+      Map(messageIdSessionParam -> MessageId().id,
+        subjectSessionParam -> Subject().subject,
+        textBodySessionParam -> TextBody().text))
+
+    feed(mailFeeder)
+      .feed(recipientFeeder)
+      .exec(sendMessage())
+  }
+
+  def sendMessage(title: RequestTitle = RequestTitle("sendMessages"),
+                  accountId: String = "accountId",
+                  username: String = "username",
+                  mailboxId: String = "draftMailboxId",
+                  messageId: String = "messageId",
+                  recipient: String = "recipient"): HttpRequestBuilder = {
+    JmapHttp.apiCall(title.title)
+      .body(StringBody(
+        s"""{
+           |  "using": ["urn:ietf:params:jmap:core","urn:ietf:params:jmap:mail"],
+           |  "methodCalls": [[
+           |    "Email/set",
+           |    {
+           |      "accountId": "$${$accountId}",
+           |      "create": {
+           |        "$${$messageId}": {
+           |          "mailboxIds": {
+           |            "$${$mailboxId}": true
+           |          },
+           |          "subject": "test mail",
+           |          "from": {"email": "$${$username}"},
+           |          "to": [{"email": "$${$recipient}"}]
+           |        }
+           |      }
+           |    },
+           |    "c1"],
+           |    ["EmailSubmission/set", {
+           |       "accountId": "$${$accountId}",
+           |       "create": {
+           |         "$SubmissionId": {
+           |           "emailId": "#$${$messageId}",
+           |           "envelope": {
+           |             "mailFrom": {"email": "$${$username}"},
+           |             "rcptTo": [{"email": "$${$recipient}"}]
+           |           }
+           |         }
+           |    }
+           |  }, "c1"]]
+           |}""".stripMargin
```

Comment:
Might need to compare again with the draft scenario... Not sure we use the params defined in mailFeeder above here for example